### PR TITLE
feat: add SavingsDaiTokenWrapper address on Ethereum

### PR DESCRIPTION
**Pull request by parth-15**

### CODE REVIEW

File: scripts/configs/pools/ethereum.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import {ChainId} from '../../generator/chains';
-import {PoolConfig} from '../types';
+import { ChainId } from '../../generator/chains';
```
Comment (miguelmtzinf) [inline scripts/configs/pools/ethereum.ts:1]: undo

File: scripts/configs/pools/ethereum.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+import { PoolConfig } from '../types';
```
Comment (miguelmtzinf) [inline scripts/configs/pools/ethereum.ts:2]: undo

File: scripts/configs/types.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import {Hex} from 'viem';
-import {ChainId} from '../generator/chains';
+import { Hex } from 'viem';
+import { ChainId } from '../generator/chains';
```
Comment (miguelmtzinf) [inline scripts/configs/types.ts]:
```suggestion
import {Hex} from 'viem';
import {ChainId} from '../generator/chains';
```

File: scripts/configs/pools/ethereum.ts (hunk 2 above)
Comment (miguelmtzinf) [inline scripts/configs/pools/ethereum.ts]:
```suggestion
import {ChainId} from '../../generator/chains';
import {PoolConfig} from '../types';
```

File: scripts/configs/types.ts (hunk 3 above)
Comment (parth-15) [reply scripts/configs/types.ts]: updated

File: scripts/configs/pools/ethereum.ts (hunk 2 above)
Comment (parth-15) [reply scripts/configs/pools/ethereum.ts]: updated

**Review by miguelmtzinf — APPROVED** [review]

**Pull request merged**